src/vos/vos_internal.h

```diff
@@ -321,7 +321,7 @@ struct vos_dtx_act_ent {
 	 */
 	daos_unit_oid_t			*dae_oids;
 	/* The time (hlc) when the DTX entry is created. */
-	daos_epoch_t			 dae_start_time;
+	uint64_t			 dae_start_time;
```

Comment:
Comment is out of date

Reply:
You are right, will refresh the patch in subsequent patch.